src/Api/Delete.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OpenEuropa\EuropaSearchClient\Api;
+
+use OpenEuropa\EuropaSearchClient\Contract\DeleteInterface;
+use OpenEuropa\EuropaSearchClient\Traits\TokenAwareTrait;
+
+class Delete extends ApiBase implements DeleteInterface
+{
+    use TokenAwareTrait;
+
+    /**
+     * @var string
+     */
+    protected $reference;
+
+    /**
+     * @inheritDoc
+     */
+    public function getConfigSchema(): array
+    {
+        return [
+            'deleteApiEndpoint' => $this->getEndpointSchema(),
```

Comment:
This will approach for having 1 url per endpoint instead of api will need to be documented, possibly in the README

Reply:
We can use `ingestionApiEndpoint` for both, ingestion & delete.